src/components/messenger/list/rewards-tooltip/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { TooltipPopup } from '../../../tooltip-popup/tooltip-popup';
+
+export interface Properties {
+  meowPreviousDayInUSD: string;
+
+  onClose: () => void;
+}
+
+interface State {
+  isTooltipOpen: boolean;
+}
+
+export class RewardsTooltip extends React.Component<Properties, State> {
+  state = {
+    isTooltipOpen: true,
+  };
+
+  componentDidMount(): void {
+    setTimeout(() => {
+      this.closeTooltip();
+    }, 3000);
+  }
+
+  closeTooltip = () => {
+    this.setState({ isTooltipOpen: false });
```

Comment:
Again not sure if we could use `showRewardsTooltip` here instead of internal state within the component?

Reply:
both are different. `showRewardsTooltip` is global state which represents "has the users seen the last day rewards or not". `isTooltipOpen` is local state which controls the open/close of tooltip rendered (in this case only `close`, since opening is handled automatically).